README.md

```diff
@@ -99,13 +100,24 @@ docker run --env-file <env_file> -p 5000:5000 --mount type=bind,source="$(pwd)"/
     Note to self ${pwd} did not work for me I got 
     docker: Error response from daemon: invalid mount config for type "bind": invalid mount path: '$(pwd)/todo_app' mount path must be absolute.
     Added docker_run_dev_mount.bat
-}
+}poert
 
 docker run --env-file <env_file> -dp 8000:8000 todo-app:prod
 
 The env_file should conform to the .env.template in the project WITHOUT QUOTES
 Example : command_line_env.env
 
+Note that the ChromeDriver needs to be in the path fro windows. 
+https://sites.google.com/a/chromium.org/chromedriver/homepoetry 
 
+Running the tests inside the docker test container
 
+Build
+docker build --target test --tag my-test-image .
 
+Run 
+docker run my-test-image todo_app/tests
+docker run my-test-image todo_app/tests_e2e
```

Comment:
I think you're missing passing through some environment variables here, e.g. with
`docker run --env-file .env my-test-image todo_app/tests_e2e` and similarly for your integration tests (as they also require the live environment variables for `test_trello.py`

Also mildly confusing that your build command here chooses a different image tag than the build command above for the same target!